apps/accelrec/app.js

```diff
@@ -1,179 +1,251 @@
+    accelx[i] = a.x * SCALE * 2; // *2 because of 8g mode
+    accely[i] = -a.y * SCALE * 2;
+    accelz[i] = a.z * SCALE * 2;
+    timestep[i] = (getTime() - timestep_start) * 1000;
+    if (accelIdx >= SAMPLES) recordStop();
 }
-function recordStart() {"ram"
-  Bangle.setLCDTimeout(0); // force LCD on
-  accelIdx = 0;
-  lastAccel = [];
-  Bangle.accelWr(0x18,0b01110100); // off, +-8g
-  Bangle.accelWr(0x1B,0x03 | 0x40); // 100hz output, ODR/2 filter
-  Bangle.accelWr(0x18,0b11110100); // +-8g
-  Bangle.setPollInterval(10); // 100hz input
-  setTimeout(function() {
-    Bangle.on('accel',accelHandlerTrigger);
-    g.clear(1).setFont("6x8",2).setFontAlign(0,0);
-    g.drawString("Waiting",g.getWidth()/2,g.getHeight()/2);
-  }, 200);
+
+function recordStart() {
+    "ram"
+    Bangle.setLCDTimeout(0); // force LCD on
+    accelIdx = 0;
+    lastAccel = [];
+    Bangle.accelWr(0x18, 0b01110100); // off, +-8g
+    Bangle.accelWr(0x1B, 0x03 | 0x40); // 100hz output, ODR/2 filter
+    Bangle.accelWr(0x18, 0b11110100); // +-8g
+    Bangle.setPollInterval(10); // 100hz input
+    setTimeout(function() {
+        Bangle.on('accel', accelHandlerTrigger);
+        g.clear(1).setFont("6x8", 2).setFontAlign(0, 0);
+        g.drawString("Waiting", g.getWidth() / 2, g.getHeight() / 2);
+    }, 200);
 }
 
 
-function recordStop() {"ram"
-  //console.log("Length:",getTime()-tStart);
-  Bangle.setPollInterval(80); // default poll interval
-  Bangle.accelWr(0x18,0b01101100); // off, +-4g
-  Bangle.accelWr(0x1B,0x0); // default 12.5hz output
-  Bangle.accelWr(0x18,0b11101100); // +-4g
-  Bangle.removeListener('accel',accelHandlerRecord);
-  E.showMessage("Finished");
-  showData();
+function recordStop() {
+    "ram"
+    //console.log("Length:",getTime()-tStart);
+    Bangle.setPollInterval(80); // default poll interval
+    Bangle.accelWr(0x18, 0b01101100); // off, +-4g
+    Bangle.accelWr(0x1B, 0x0); // default 12.5hz output
+    Bangle.accelWr(0x18, 0b11101100); // +-4g
+    Bangle.removeListener('accel', accelHandlerRecord);
+    E.showMessage("Finished");
+    showData();
 }
 
 
 function showData() {
-  g.clear(1);
-  var w = g.getWidth()-20; // width
-  var m = g.getHeight()/2; // middle
-  var s = 12; // how many pixels per G
-  g.fillRect(9,0,9,g.getHeight());
-  g.setFontAlign(0,0);
-  for (var l=-8;l<=8;l++)
-    g.drawString(l, 5, m - l*s);
-
-  function plot(a) {
-    g.moveTo(10,m - a[0]*s/SCALE);
-    for (var i=0;i<SAMPLES;i++)
-      g.lineTo(10+i*w/SAMPLES, m - a[i]*s/SCALE);
-  }
-  g.setColor("#0000ff");
-  plot(accelz);
-  g.setColor("#ff0000");
-  plot(accelx);
-  g.setColor("#00ff00");
-  plot(accely);
-
-  // work out stats
-  var maxAccel = 0;
-  var tStart = SAMPLES, tEnd = 0;
-  var vel = 0, maxVel = 0;
-  for (var i=0;i<SAMPLES;i++) {
-    var a = accely[i]/SCALE;
-    if (a>0.1) {
-      if (i<tStart) tStart=i;
-      if (i>tEnd) tEnd=i;
+    g.clear(1);
+    let csv_files_N = require("Storage").list(/^acc.*.csv$/).length;
```

Comment:
You probably want this, so it matches the dot literally:
```suggestion
    let csv_files_N = require("Storage").list(/^acc.*\.csv$/).length;
```

(and below)

Reply:
Thanks! I will fix this!